### ISSUE TITLE
ReST syntax misparses ``makeService`` ,

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6902]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)| @lvh reported|
|-|-|
|Trac ID|trac#6902|
|Type|defect|
|Created|2014-01-16 10:14:46Z|
|Branch|https://github.com/twisted/twisted/tree/fix-makeService-reST-markup-6902|

See: https://twisted.readthedocs.org/en/latest/projects/core/howto/tutorial/library.html

`````makeService`` ,``` occurs literally in the compiled source; it should of course look more like ```makeService```,.

<details><summary>Searchable metadata</summary>

```
trac-id__6902 6902
type__defect defect
reporter__lvh lvh
priority__normal normal
milestone__ 
branch__branches_fix_makeService_reST_markup_6902 branches/fix-makeService-reST-markup-6902
branch_author__hynek hynek
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1389867286000000 1389867286000000
changetime__1389872350000000 1389872350000000
version__None None
owner__hynek hynek

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_2"></a>|@hynek commented|
|-|-|

(In [41262]) Branching to fix-makeService-reST-markup-6902.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_3"></a>|@hynek commented|
|-|-|

The problem was a missing space in front of the markup (and a surplus space after it).

I’ve fixed it and took the artistic liberty to use better reST markup in the adjacencies + wordsmith some of the text which was awkward in parts. And removed lots of empty lines.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

I tried to review this but building the docs successfully seems to require a particular version of Sphinx (different from the version I have and not documented so far as I can tell).  BuildBot's version also fails to build the docs (though, of course, in the tradition of reST and Sphinx it fails by claiming it succeeded and generating garbage output).

**Comment by twisted-trac** [comment]
|[<img alt="dstufft's avatar" src="https://avatars.githubusercontent.com/u/145979?s=50" width="50" height="50">](https://github.com/dstufft)|@dstufft set owner to @dstufft<br>@dstufft set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="dstufft's avatar" src="https://avatars.githubusercontent.com/u/145979?s=50" width="50" height="50">](https://github.com/dstufft)<a name="note_6"></a>|@dstufft set owner to @hynek<br>@dstufft set status to `new`|
|-|-|

LGTM and it builds locally for me using Sphinx 1.2

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_7"></a>|@hynek set status to `closed`|
|-|-|

(In [41265]) Merge fix-makeService-reST-markup-6902

Author: hynek
Reviewer: dstufft
Fixes: [#6902](https://github.com/twisted/twisted/issues/6902)

Fix a markup error and polish the surroundings.
